charts/rancher-eks-controller/0.1.000/templates/eksclusterconfig.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: eksclusterconfigs.ke.cattle.io
+spec:
+  group: ke.cattle.io
+  names:
+    kind: EKSClusterConfig
+    plural: eksclusterconfigs
+    shortNames:
+    - ekscc
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          anyOf:
+          - required:
+            - imported
+            - cloudCredential
+            - displayName
+          - required:
+            - publicAccess
+            - privateAccess
+            - cloudCredential
+            - displayName
+          properties:
+            cloudCredential:
+              type: string
+            displayName:
+              type: string
+            imported:
+              enum:
+              - true
+              nullable: true
+              type: boolean
+            kmsKey:
+              type: string
+            kubernetesVersion:
+              type: string
+            loggingTypes:
+              items:
+                type: string
+              nullable: true
+              type: array
+            nodeGroups:
+              items:
+                properties:
+                  desiredSize:
+                    nullable: true
+                    type: integer
+                  diskSize:
+                    nullable: true
+                    type: integer
+                  ec2SshKey:
```

Comment:
Is the ssh key still a thing ?

Reply:
yes it is, it still uses ec2 node which you can ssh into